nc/source/audio/AudioModuleImpl.cpp

```diff
@@ -22,14 +23,20 @@ namespace
          *  quickly see when they happen. */
         if(status) std::cerr << "Audio stream underflow\n";
 
-        auto* system = static_cast<nc::audio::AudioSystemImpl*>(userData);
+        auto* system = static_cast<nc::audio::AudioModuleImpl*>(userData);
         return system->WriteToDeviceBuffer(static_cast<double*>(outputBuffer));
     }
 }
 
 namespace nc::audio
 {
-    AudioSystemImpl::AudioSystemImpl(Registry* registry)
+    auto BuildAudioModule(Registry* reg) -> std::unique_ptr<AudioModule>
+    {
+        /** @note If we make a stub version of this, consider the cast in AudioSystemCallback. */
```

Comment:
Make this a todo with a github issue. We may need to stub this for testing on github runners.